content/projects/data-science-specific/bootcamp-exploratory-data-analysis/_index.md

```diff
@@ -0,0 +1,55 @@
+---
+content_type: project
+flavours:
+- python
+prerequisites:
+  hard:
+  - topics/data-science-specific/datacamp/intro-to-python
+  soft:
+  - topics/data-science-specific/intro-to-google-colab
+ready: true
+submission_type: repo
+title: Bootcamp Exploratory Data Analysis
+---
+
+You will need to download and add all the relevant files from [here](https://drive.google.com/drive/folders/1scmLfz6SswLqKoj1S6s3VytezCR7m_dJ?usp=sharing) and add them to the repo supplied.
+
+Your repo should contain everything needed to replicate your work:
+- data
+- notebook
+- `requirements.txt`
+
+## Instructions
+
+Your task is to use the given data to answer the questions found in `questions.md`. Any other additional analysis you think will help your submission is cool with us but NO external data may be used.
```

Comment:
Rather just write the questions here instead of linking somewhere else. Is there a reason to not use version control on our questions?

Reply:
So the idea Sam wanted us to go with was that instead of having one (easily searchable) project for bootcamp there would be multiple that we "cycled" through for each bootcamp.

All that would need updating would be the data and questions.